feature_tests/kotlin/somelib/src/main/java/dev/gigapixel/somelib/DiplomatStr.java

```diff
@@ -0,0 +1,16 @@
+package dev.gigapixel.somelib;
+import com.sun.jna.Pointer;
+import com.sun.jna.Structure;
+
+import java.util.List;
+
+public class DiplomatStr extends Structure implements Structure.ByValue {
```

Comment:
This is not used yet, but will be need to implement string slice compatibility, and is a prototype for struct types via jna that I'd like to keep around